protocol/messenger_contact_verification.go

```diff
@@ -962,6 +962,24 @@ func (m *Messenger) HandleDeclineContactVerification(state *ReceivedMessageState
 		return nil // Do nothing, We have already cancelled the verification request
 	}
 
+	contact.VerificationStatus = VerificationStatusUNVERIFIED
```

Comment:
Not sure about this. Identity verification flows are not bi-directional, and we already mark contacts as unverified in  [CancelVerificationRequest](https://github.com/status-im/status-go/blob/develop/protocol/messenger_contact_verification.go#L182). But maybe requirements have changed and i'm not aware of it

Reply:
You're right, I forgot to think about the flows being bi-directional.

Good thing for me though, this is still correct as far as I understand.

We use `receivedVerifications` for the verifications that comes from other users. And we use the `contact.VerificationStatus` for the ones we send.

So in this case, we sent the verification to user B, so we marked the `VerificationStatus` as `VerificationStatusVERIFYING`. 
Then user B canceled, so the status goes back to the default one of `VerificationStatusUNVERIFIED`.

This doesn't affect the verification status from recived verifications. If user B sent us a verification and we sent back a response and they approved it, we would be `VerificationStatusVERIFIED` for them.